dts/arm/nxp/nxp_imx8m_m4.dtsi

```diff
@@ -178,6 +178,36 @@
 					       RDC_DOMAIN_PERM_RW))>;
 			status = "disabled";
 		};
+
+		ecspi1: spi@30820000 {
```

Comment:
Would you please reorder the nodes in address?

Reply:
Done.
Maybe this should also be done for `nxp_imx8ml_m7.dtsi`? There are nodes at the end that are not ordered in addresses.

Reply:
yes, we should, but this PR has no relation with it, unless a PR to update it.